Makefile

```diff
@@ -10,9 +10,9 @@ IOS_SIMULATOR_ARCHIVE_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/Products/
 IOS_SIMULATOR_ARCHIVE_DSYM_PATH = $(CURR_DIR)/build/ios_simulator.xcarchive/dSYMs/
 IOS_ARCHIVE_PATH = $(CURR_DIR)/build/ios.xcarchive/Products/Library/Frameworks/
 IOS_ARCHIVE_DSYM_PATH = $(CURR_DIR)/build/ios.xcarchive/dSYMs/
-TVOS_SIMULATOR_ARCHIVE_PATH = ./build/tvos_simulator.xcarchive/Products/Library/Frameworks/
+TVOS_SIMULATOR_ARCHIVE_PATH = $(CURR_DIR)./build/tvos_simulator.xcarchive/Products/Library/Frameworks/
 TVOS_SIMULATOR_ARCHIVE_DSYM_PATH = $(CURR_DIR)/build/tvos_simulator.xcarchive/dSYMs/
-TVOS_ARCHIVE_PATH = ./build/tvos.xcarchive/Products/Library/Frameworks/
+TVOS_ARCHIVE_PATH = $(CURR_DIR)./build/tvos.xcarchive/Products/Library/Frameworks/
```

Comment:
do these paths need to change? I did not see this in core

Reply:
Just wanted to update to use a common pattern, either use `./build` or `${CURR_DIR}/build`. 

@cacheung  remove . after ${CURR_DIR}
` ${CURR_DIR}/build`